ddtrace/tracer.py

```diff
@@ -73,15 +77,20 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None):
         if sampler is not None:
             self.sampler = sampler
 
-    def trace(self, name, service=None, resource=None, span_type=None):
-        """Return a span that will trace an operation called `name`.
+    def trace(self, name, service=None, resource=None, span_type=None, ctx=None, span_parent=None):
```

Comment:
Nice, happy to see that OT is doing it that way ; I'd vote for relying on the parent_span only too.